api/rest/client.go

```diff
@@ -62,16 +63,21 @@ func (d DiagnosticsClient) Create(ctx context.Context, node string, id string) (
 	defer resp.Body.Close()
 
 	if resp.StatusCode != http.StatusOK {
-		return nil, fmt.Errorf("unexpected HTTP status code received: %d", resp.StatusCode)
+		body, e := ioutil.ReadAll(resp.Body)
```

Comment:
Umm, no. We're reading the full body here just to cut off everything beyond the first 100 characters. Why not read only the first 100 bytes to begin with?

Reply:
Thanks, fixed